.github/workflows/build-and-test.yaml

```diff
@@ -1,72 +1,27 @@
-on:
-  push:
-  pull_request:
-    branches:
-      - master
-  workflow_dispatch:
+name: Build
 
-jobs:
-  build:
```

Comment:
Why remove the non-wheels build?

Perhaps it becomes redundant afterwards, but reducing the number of changes in this PR would also make it easier for project maintainers review it.